server/group/Settings of Server/client.js

```diff
@@ -0,0 +1,20 @@
+const { Client } = require("pg")
+require('dotenv').config()
+
+const client = new Client(
+    process.env.DATABASE_URL
+        ? {
+            connectionString: process.env.DATABASE_URL,
+            ssl: {
+                rejectUnauthorized: false
+            }
+        }
+        : {
+            user: process.env.DB_USER,
+            host: process.env.DB_HOST,
+            database: process.env.DB_NAME,
+            password: process.env.DB_PSW,
+            port: process.env.DB_PORT
+        }
+);
+module.exports.client = client
```

Comment:
please rename folder 'Settings of Server' to something without spaces (f.e. settingsOfServer).

And this code is not just settings. It is piece of code which connects to database.

Reply:
So it's better to name folder like 'loaders' instead of 'settingsOfServer'